src/content/docs/en/guides/deploy/cloudflare.mdx

```diff
@@ -137,3 +137,14 @@ If you're encountering errors, double-check the version of `node` you're using l
 Cloudflare requires [Node v16.13](https://miniflare.dev/get-started/cli#installation), which is a more recent version than Astro’s out-of-the-box minimum, so double check you’re using at least v16.13.
 
 Client-side hydration may fail as a result of Cloudflare's Auto Minify setting. If you see `Hydration completed but contains mismatches` in the console, make sure to disable Auto Minify under Cloudflare settings.
+
+If you are deploying or building a project that is using server-side rendering and the server fails to build with error messages similar to:
+
+`[Error] Could not resolve "XXXX` `The package "XXXX" wasn't found on the file system but is built into node. Are you trying to
+  bundle for node? You can use "platform: 'node'" to do that, which will remove this error.`
+
+This means that a package or import you are using in the server-side environment is not compatible with the Cloudflare node specification. 
```

Comment:
> Cloudflare node specificiation

Not sure about the naming, also it would be great to add a link to the Cloudflare docs about this.

Reply:
```suggestion
This means that a package or import you are using in the server-side environment is not compatible with the [Cloudflare Workers runtime APIs](https://developers.cloudflare.com/workers/runtime-apis/nodejs/). 
```

Reply:
In the Cloudflare docs they refer to their implementations of the node.js runtime as the Workers runtime API. Hopefully this is a more apt name.